rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -140,3 +147,43 @@ def test_last_queried_ts(tokens, freezer):
         assert len(after_second_query) == 1
         assert after_second_query[0][0] == 'last_queried_timestamp'
         assert int(after_second_query[0][1]) >= continuation
+
+
+def test_cache_is_per_token_type(ethereum_inquirer):
+    """This test makes sure that tokens info cache is per token type"""
+    def query_token_info(token_kind):
+        return _query_or_get_given_token_info(
```

Comment:
can you add a small comment explaining how this is used later with the patch to return the desired response?